test/cpp/qps/qps_json_driver.cc

```diff
@@ -49,10 +49,103 @@ DEFINE_string(scenarios_file, "",
 DEFINE_string(scenarios_json, "",
               "JSON string containing an array of Scenario objects");
 DEFINE_bool(quit, false, "Quit the workers");
+DEFINE_bool(
+    search, false,
+    "Search for the offered_load value that achieves targeted cpu load");
+DEFINE_double(initial_offered_load, 1000.0,
+              "Set up for intial offered load to start the search");
+DEFINE_double(targeted_cpu_load, 99.0, "Targeted cpu load");
+DEFINE_double(precision, 500, "Final search result precision");
```

Comment:
The precision defines the threshold for the search range ([low, high]) that below which, we will stop the search. Unit is the same as the search param. For the case of offered_load, this can save a lot of time by setting the precision around 500, since 1% change in cpu_load  may need 1000+ change in offered_load value and the variation is usually higher than 1%.